rllib/policy/policy.py

```diff
@@ -164,6 +166,38 @@ def compute_single_action(self,
         return action, [s[0] for s in state_out], \
             {k: v[0] for k, v in info.items()}
 
+    @DeveloperAPI
```

Comment:
Add this new method to Policy API will simplify and unify across different Algos:
1) log-likelihood calculations.
2) Action sampling.
3) Exploration handling.